### PR TITLE
add slim scroll to battle results

### PR DESCRIPTION
**Pull request by mklemersson**
add a scrollbar to battle results at battle page

### CODE REVIEW

File: resources/views/battle-results.blade.php [hunk 1]
```diff
@@ -68,4 +69,17 @@
             margin-bottom: 10px;
         }
     </style>
-@endsection
\ No newline at end of file
+@endsection
+
+@section("scripts")
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jQuery-slimScroll/1.3.8/jquery.slimscroll.min.js"></script>
```
Comment (hudsonpereira) [inline resources/views/battle-results.blade.php:75]: Could you insert this lib as a npm dependency?

Also, i think this one is a little bit more popular:

https://github.com/malihu/malihu-custom-scrollbar-plugin

File: resources/views/battle-results.blade.php [hunk 2]
```diff
@@ -68,4 +69,17 @@
...
+    <script type="text/javascript">
+        $(function () {
+            $('#battle-results').slimScroll({
+                height: (73 * 5) + 'px',
```
Comment (hudsonpereira) [inline resources/views/battle-results.blade.php:79]: Why is it 365?

Comment (mklemersson) [reply resources/views/battle-results.blade.php:79]: It's minimum height, like when we add the 5th panel, the scroll bar will shown, understand?

File: resources/views/battle-results.blade.php (hunk 1 above)
Comment (mklemersson) [reply resources/views/battle-results.blade.php:75]: It's possible

**Comment by mklemersson** [comment]
I'm closing this PR because the next OR also will contain this work ;)

**Pull request closed without merging**